packages/components/molecules/f-alert/src/components/_tests/Alert.test.js

```diff
@@ -164,4 +164,42 @@ describe('Alert', () => {
             });
         });
     });
+
+    describe('ariaLiveAttribute', () => {
```

Comment:
do we want to test for the presence of the actual attribute in the html as well?

Reply:
I imagine if the attribute was missing both tests would fail

Reply:
I did go down the component test route, but it didn't look obvious how to provide different props, ill have another look and see.